app/controllers/facility_reservations_controller.rb

```diff
@@ -141,7 +141,7 @@ def create
   # GET /facilities/:facility_id/instruments/:instrument_id/reservations/:id/edit_admin
   def edit_admin
     @instrument  = current_facility.instruments.find_by_url_name!(params[:instrument_id])
-    @reservation = @instrument.reservations.find(params[:reservation_id])
+    @reservation = Reservation.where(product_id: @instrument.schedule.products.map(&:id)).find(params[:reservation_id])
```

Comment:
I think this can be written more simply as `@instrument.schedule.reservations.find`. That's what `InstrumentController#schedule` uses. We might also want to add the `non_user` scope here to prevent modifying actual orders.